Cargo.toml

```diff
@@ -19,3 +19,6 @@ codegen-units = 1
 lto = true
 panic = "abort"
 
+[patch.crates-io]
+# Using this patch until a version > 4.0 gets published.
```

Comment:
Can we mark it with a `TODO` instead so that we can keep track of it?

Reply:
done